packages/firestore/lib/FirestoreQuery.js

```diff
@@ -437,7 +437,11 @@ export default class FirestoreQuery {
       );
     }
 
-    if (isNull(value) && !this._modifiers.isEqualOperator(opStr)) {
+    if (
+      isNull(value) &&
+      !this._modifiers.isEqualOperator(opStr) &&
+      !this._modifiers.isNotEqualOperator(opStr)
```

Comment:
This *does* in fact appear valid + correct. `!=` is supported in comparisons against null values, so this change looks good